### PR TITLE
Streamline JDBC source tests (live)

### LINKED ISSUE #437: Fix Neural net tests so that running against non VA-enabled server doesn't loop infinitely.

### PR DESCRIPTION
**Pull request by fhcarter**
Fixes #437 

No need to constantly recreate the source & session.  Speeds up tests, etc.

### CODE REVIEW

**Comment by fhcarter** [comment]
(Also fixes infinite loop in OR tests so that it completes if timed out.

**Comment by fhcarter** [comment]
[Jenkins run](https://jenkins.vantiq.com/jenkins/job/vantiq-extension-sources-branch/195/) -- clean

File: jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java [hunk 1]
```diff
@@ -353,229 +352,254 @@ public static void tearDown() throws VantiqSQLException {
...
+            try {
+                queryResult = jdbc.processPublish(PUBLISH_QUERY);
+                assert queryResult > 0;
+            } catch (VantiqSQLException e) {
+                fail("Should not throw an exception: " + e.getMessage());
+            }
+        } finally {
+            if (jdbc != null) {
+                jdbc.close();
+            }
```
Comment (AlexBlumer) [inline jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java:387]: It looks like most of the "changes" are just indentation due to this. Could you just do an `@After` instead with this line?

**Review by AlexBlumer — APPROVED** [review]
LGTM

File: jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java (hunk 1 above)
Comment (fhcarter) [reply jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java:387]: Yes -- I thought the async stuff was different, but it's just unused there.  So @Before & @After simplifies things further.

**Pull request merged**